dags/quoteday.py

```diff
@@ -0,0 +1,26 @@
+import random
+
+import pendulum
+from airflow import DAG
+# from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+# from airflow.operators.python import BranchPythonOperator
+# from airflow.utils.trigger_rule import TriggerRule
+
+import msg2teams
+
+def _func():
```

Comment:
What is this?

Reply:
Oh, I thought that I would need Branching operator, so I used the template from the lectures. It is not needed, but just in case I left it here. Can be removed for now.